File: 프로그래머스/0/181902. 문자 개수 세기/문자 개수 세기.js
function solution(my_string) {
    let arr = new Array(52).fill(0);

    for (let char of my_string) {
        let code = char.charCodeAt(0);
        if (code >= 65 && code <= 90) {
            arr[code - 65] += 1;
        } else if (code >= 97 && code <= 122) {
            arr[code - 71] += 1;
        }
    }

    return arr;
}

File: 프로그래머스/0/181902. 문자 개수 세기/문자 개수 세기.java
class Solution {
    public int[] solution(String my_string) {
        int[] arr = new int[52];
        
        for (char c : my_string.toCharArray()) {
            if ('A' <= c && 'Z' >= c) {
                arr[c - 65]++;
            } else {
                arr[c - 97 + 26]++;
            }
        }
        
        return arr;
    }
}